### ISSUE TITLE
[cloud-tensorflow-mnist]: Typo in example in #5

### ISSUE THREAD

**Issue by ianmcook** [googlecodelabs/feedback#371]
@martin-gorner In https://codelabs.developers.google.com/codelabs/cloud-tensorflow-mnist/#5, this line:
`a,c = sess.run([accuracy, cross_entropy], feed=test_data)`
should be:
`a,c = sess.run([accuracy, cross_entropy], feed_dict=test_data)`
(the `_dict` is missing)
